script/Ciel/Day 14/14_01B_01_CIEL14B_1.txt

```diff
@@ -1,67 +1,74 @@
+I don't know if it's because she isn't paying attention or what,
+ but it seems like Arcueid's perfection is fading away.
 }
 [sha:4356e07472646777d42319e4a7abdcb7e6e0e8bc]{
 -- Page 1, Offset 38315.
 -- 懇願するような声が<囁|ささや>かれる。
-She whispers out in a plea-like voice.
+Her pleading voice is little more than a whimper.
 }
 [sha:c34559effdb31d717759ae533f562fe0da2855c6]{
 -- Page 2, Offset 38316.
 -- ……だからこそ聞けない。
-...But I can't listen to her.
+// LintConsistency:37604
+But... I can't answer her.
 }
 [sha:5f1251eeaf2595c6383df3ae9cf484e853a1b7e8]{
 -- Page 2, Offset 38317.
 -- 俺は今の彼女から目を背ける事はできない。反感からではなく、俺はその間違いを、どうしても見逃せない。
-I can't turn my eyes away from her now. It's not out of animosity, but rather, because I can't look past that mistake no matter what.
+I can't turn away from her now.
+ Not because I hate her――but because I absolutely cannot let that mistake go uncorrected.
 }
 [sha:2866ce5ce24b7460d72948a8596642cc88932a22]{
 -- Page 3, Offset 38318.
 -- 「……志貴。それとも、わたしのことは嫌い？」
-"...Shiki. Could it be that you hate me?"
+"Shiki... Could it be that you hate me?"
 }
 [sha:b30dd908cc46da1e0e79d56f571a7ff1a8b9c59c]{
 -- Page 3, Offset 38319.
 -- まっすぐに見つめてくる。
-She looks straight into me.
+She stares straight at me.
 }
 [sha:1abd07ce1ce333caffecf5630a44d4e1a65c18b4]{
 -- Page 3, Offset 38320.
 -- 押さえつけられながら彼女の顔を見つめ返す。
-I stare back at her face as I'm being pinned down.
+Still pinned to the ground, I gaze back up at her face.
 }
 [sha:2a51c6c39be172629a11337692fddc193f1824d4]{
 -- Page 3, Offset 38321.
 -- 真摯に。―――脳が焼き切れてしまうほど、強く。
-In all sincerity. A sincerity strong enough to fry my brain.
+I fixate on it.
+ I stare at it so hard I feel like my brain's about to burst into flames.
 }
 [sha:b48253753a34f44f087a75e031cd0c7001941669]{
 -- Page 3, Offset 38322.
 -- どうしても出来なかった／視たくなかった、その個所を。
-I desperately search for that spot that I never wanted to see.
+I desperately hunt for the point I couldn't see――that I never wanted to see.
 }
 [sha:90c7826f4e76ff196fcabf34d19d8c47d61dc127]{
 -- Page 4, Offset 38323.
 -- ……びきり、と頭の芯で音がする。
-...Creak. The core of my head makes a sound.
+I hear the inner workings of my brain creak under the pressure.
 }
 [sha:ce37877a6cb2aeb1123880b16aa2e03bc5f503dd]{
 -- Page 4, Offset 38324.
 -- 今までのように頭蓋骨を割る音ではなく―――脳そのものに亀裂が走ってしまいそうな音。
-It's not the sound of my skull being cracked until now―――but rather, the sound of fissures running through the brain itself.
+It isn't the sound of my skull cracking――I've heard that before.
+ This is the sound of fissures running through my brain itself.
 }
 [sha:371f28ce66ab60fb667774cda1fd356705ca6830]{
 -- Page 5, Offset 38325.
 -- ……もうこの先、これ程の静かさはないという程に、アルクェイドの体を凝視する。
-...From here on out, I stare into Arcueid's body silently like never before.
+// This one did not land for me, needs a good check
+I examine Arcueid's body intently,
+ as though there will never again be a time that she's this silent.
 }
 [sha:4b9779c1adb69fb6c716b9a1b769de817cef96d1]{
 -- Page 6, Offset 38326.
```

Comment:
>I can't narrow it down by looking at all of her at once.

38327
Well, that's not right. Not an easy one, but nevertheless.

Clauses moved around a bit.
>I need to look at just one point and etch it into my mind.
>The point between her consciousness and her mind——
>the point where she most likely holds together the weakest.